style.css

```diff
@@ -0,0 +1,268 @@
+.body {
+   margin: 0;
+ }
+
+ .header {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)